### ISSUE TITLE
o.seal.quips.get is not a function

### ISSUE THREAD

**Issue by polneiman** [tloncorp/landscape-apps#2336]
```
TypeError: o.seal.quips.get is not a function
    at https://sortud-nopfel.tlon.network/apps/groups/assets/index.8af1b904.js:36:66683
    at Object.useMemo (https://sortud-nopfel.tlon.network/apps/groups/assets/react-beautiful-dnd.3bfe057f.js:19:58807)
    at f.useMemo (https://sortud-nopfel.tlon.network/apps/groups/assets/@tlon/sigil-js.69e385bd.js:12:5789)
    at Kf (https://sortud-nopfel.tlon.network/apps/groups/assets/index.8af1b904.js:36:66658)
    at yN (https://sortud-nopfel.tlon.network/apps/groups/assets/index.8af1b904.js:48:228438)
    at Kl (https://sortud-nopfel.tlon.network/apps/groups/assets/react-beautiful-dnd.3bfe057f.js:19:52884)
    at mf (https://sortud-nopfel.tlon.network/apps/groups/assets/react-beautiful-dnd.3bfe057f.js:23:8723)
    at pf (https://sortud-nopfel.tlon.network/apps/groups/assets/react-beautiful-dnd.3bfe057f.js:23:947)
    at yg (https://sortud-nopfel.tlon.network/apps/groups/assets/react-beautiful-dnd.3bfe057f.js:23:875)
    at rn (https://sortud-nopfel.tlon.network/apps/groups/assets/react-beautiful-dnd.3bfe057f.js:23:730)
```

**Comment by jamesacklin** [comment]
@polneiman Thanks. Do you happen to remember which notebook you were in when this error occurred?

**Comment by polneiman** [comment]
not really.  since 6 days i couldn't join the group.  I just wanted to do
it from the "Groups" app and I got this error.  The striking thing is that
after sending the error I was able to join!

**Comment by jamesacklin** [comment]
Interesting and glad you got in. You joined the group via a note embedded in a chat, or a group link embedded in a note?

**Comment by polneiman** [comment]
A group link embedded in a note.

**Comment by jamesacklin** [comment]
Thank you, will investigate.

**Comment by arthyn** [comment]
Should be fixed in #2309 going out with 3.0 soon
